### DISCUSSION TITLE
plan synapps avec composite

### DISCUSSION THREAD

**Discussion by Kabech** [witsa/synapps#726] [Q&A] [answered]
bonjour; j'essaie   i de faire un plan avec les composite que J'ai crée mais a chaque fois j ajoute les composite dans le plan(image) ils se redimensionnes lorsque j exécute.
merci a l'avance.

**Comment by Raphwit** [comment, marked as answer]
Bonjour, si vous utilisez une boite à vue, celle-ci adapte la taille de son contenu à la taille du contenant, si la boite à vue change de taille, l'image va automatiquement être retaillée, il faut donc être attentif à ce détail lorsque l'on ajoute des composites dans des boites à vue.

Il faut également faire attention aux tailles que l'on a renseignées dans les composites qui peuvent être relatives au contenant et donc varier en fonction de la taille du contenant.

Pour ce type d'utilisation, il est conseillé d'utiliser plutôt une toile au sein d'une boite à vue pour disposer les éléments et qu'ils s'adaptent automatiquement.

La structure pourrait ressembler à quelques chose comme cela : 

![image](https://github.com/witsa/synapps/assets/93767525/ba75e5a4-8009-4918-b257-19019dea3ef0)

Les acteurs seraient donc dans une toile et s'adapterais automatiquement grâce à la boite à vue.

**Comment by Kabech** [comment]
Merci pour le retour,
Est ce que vous me conseillez de faire comme le composi vanne ?
Pour récupérer l'état d'une lampe et enfin la contrôler depuis le plan !!
@Raphwit
